### PR TITLE
Added in-built media player to play video without navigating to another window

### LINKED ISSUE #56: Video/Image not available to preview
**Describe the bug**
The video is not playing on the same page https://space-voyager.vercel.app/apod

**To Reproduce**
1. Go to 'https://space-voyager.vercel.app/apod'
2. Scroll down to the footer

**Expected behavior**
The video should be playing on the same page, rather than navigating to another window

**Screenshots**
image

### PR DESCRIPTION
**Pull request by ramdevcm**
image

### CODE REVIEW

**Comment by mohittk** [comment]
@ramdevcm I don't quite get it what you are willing to add? Before making any off-topic pull request. Please do create issue.

**Comment by ramdevcm** [comment]
can I know why this is labelled as invalid?

**Comment by mohittk** [comment]
> can I know why this is labelled as invalid?

Deployment got failed. and in addition to it, you did'nt created any issue for this.

**Comment by umgbhalla** [comment]
> can I know why this is labelled as invalid?

because it fails ci deployment on vercel

**Comment by ramdevcm** [comment]
image

This was the issue

**Comment by ramdevcm** [comment]
image/video not played/displayed

**Comment by ramdevcm** [comment]
should I need to create an issue?

**Comment by mohittk** [comment]
> should I need to create an issue?

Yes! you should. It will be better, and check what error you got while deploying

**Comment by umgbhalla** [comment]
@mohittk copy paste deployment [logs](https://vercel.com/mohit-space/space-voyager/E9kZ91unpwR1aovctXqiDjEiC1cr) here , its setup on your account hence only you have access

**Comment by mohittk** [comment]
@ramdevcm checkout logs right here.
13:51:13.738  	Cloning github.com/space-voyager-21/space-voyager (Branch: master, Commit: 7915063)
13:51:14.576  	Cloning completed: 837.411ms
13:51:14.601  	Analyzing source code...
13:51:15.908  	Installing build runtime...
13:51:18.322  	Build runtime installed: 2.414s
13:51:20.823  	Looking up build cache...
13:51:23.356  	Build cache downloaded [32.94 MB]: 2312.579ms
13:51:24.625  	Detected package.json
13:51:24.625  	Installing dependencies...
13:51:24.936  	yarn install v1.22.11
13:51:25.038  	[1/4] Resolving packages...
13:51:25.819  	success Already up-to-date.
13:51:25.826  	Done in 0.90s.
13:51:26.363  	> space-app@0.1.0 build /vercel/path0
13:51:26.363  	> react-scripts build
13:51:27.977  	Creating an optimized production build...
13:51:30.782  	Failed to compile.
13:51:30.783  	./src/components/apod/Apod.js
13:51:30.783  	Cannot find module: 'react-player/lazy'. Make sure this package is installed.
13:51:30.784  	You can install this package by running: yarn add react-player/lazy.
13:51:30.799  	npm ERR! code ELIFECYCLE
13:51:30.799  	npm ERR! errno 1
13:51:30.802  	npm ERR! space-app@0.1.0 build: `react-scripts build`
13:51:30.803  	npm ERR! Exit status 1
13:51:30.803  	npm ERR! 
13:51:30.803  	npm ERR! Failed at the space-app@0.1.0 build script.
13:51:30.803  	npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
13:51:30.816  	npm ERR! A complete log of this run can be found in:
13:51:30.816  	npm ERR!     /vercel/.npm/_logs/2021-10-10T08_21_30_803Z-debug.log
13:51:30.831  	Error: Command "CI=false npm run build" exited with 1

**Comment by umgbhalla** [comment]
ig he didnt push package.json

**Comment by mohittk** [comment]
> ig he didnt push package.json

Are you sure? you mean any problem in dependencies?

**Comment by umgbhalla** [comment]
> > ig he didnt push package.json
> 
> Are you sure? you mean any problem in dependencies?

13:51:30.783 Cannot find module: 'react-player/lazy'. Make sure this package is installed.
13:51:30.784 You can install this package by running: yarn add react-player/lazy.

😑

**Comment by ramdevcm** [comment]
yes
added a new package

**Comment by mohittk** [comment]
> > > ig he didnt push package.json
> > 
> > 
> > Are you sure? you mean any problem in dependencies?
> 
> 13:51:30.783 Cannot find module: 'react-player/lazy'. Make sure this package is installed. 13:51:30.784 You can install this package by running: yarn add react-player/lazy.
> 
> 😑

Ohh yea! here it is.

**Comment by umgbhalla** [comment]
> yes added a new package

then commit and push package.json   **without**   closing this pull request

**Comment by ramdevcm** [comment]
can you add it??
or should I PR again?

**Comment by ramdevcm** [comment]
OK
@umgbhalla

**Comment by umgbhalla** [comment]
u dont need to close this pr , just commit and push @ramdevcm

File: package.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-particles-js": "^3.5.3",
-    "react-player": "^2.9.0",
```
Comment (ramdevcm) [inline package.json]: change in dependency

**Comment by ramdevcm** [comment]
Code Pushed

**Review by umgbhalla — APPROVED** [review]

**Comment by umgbhalla** [comment]
@mohittk you can merge now

**Comment by mohittk** [comment]
@ramdevcm 
You did it finally!

**Comment by ramdevcm** [comment]
Hehee... Thank you, Guys

**Pull request merged**